fe/src/App.jsx

```diff
@@ -1,26 +1,14 @@
 import React from 'react';
-import logo from './logo.svg';
+import { Route, Routes } from 'react-router-dom';
 import './App.css';
+import DashboardPage from './pages/user/dashboard/DashboardPage';
+import Register from './pages/user/register/Register';
```

Comment:
- small improvements: follow proper order of imports **(apply to all related)**
```
installed_packages_here

your_made_files_here

css_here
```

- in your case.

```
import React from 'react';
import { Route, Routes } from 'react-router-dom';

import DashboardPage from './pages/user/dashboard/DashboardPage';
import Register from './pages/user/register/Register';

import './App.css';
```